src/jobQueue.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  run,
+  quickAddJob,
+} from 'graphile-worker';
+import { processBulkUpload } from './tasks';
+import type { ProcessBulkUploadJobPayload } from './types';
+
+enum JobType {
+  PROCESS_BULK_UPLOAD = 'processBulkUpload',
+}
+
+const getJobQueueConnectionString = () => (
+  `postgres://${process.env.PGUSER}:${process.env.PGPASSWORD}@${process.env.PGHOST}:${process.env.PGPORT}/${process.env.PGDATABASE}`
```

Comment:
This is going to break my local workflow, as I don't have all those environment variables set. Fortunately, I don't believe you need to set the connection string at all - as long as either the `PGDATABASE` or `DATABASE_URL` environment variables are set, the underlying pg-node should pick up its configuration automatically:

> One of these must be provided (in order of priority):
>
> - `pgPool` pg.Pool instance
> - `connectionString` setting
> - `DATABASE_URL` envvar
> - [PostgreSQL environmental variables](https://www.postgresql.org/docs/current/libpq-envars.html), including at least `PGDATABASE` (NOTE: not all envvars are supported)

https://worker.graphile.org/docs/library/run#runneroptions